### DISCUSSION TITLE
SVG Animation Output (asked before but i want to reask)

### DISCUSSION THREAD

**Discussion by ghost** [friction2d/friction#117] [Q&A] [answered]
I want to just SVG outputs for my [site](https://alnes.codeberg.page/).
Please help me, guys.
And please mirror friction to Codeberg for more wide audience.

**Comment by rodlie** [comment, marked as answer]
> I want to just SVG outputs for my site

File=>Export=>Web Animation (`Shift+F12`)

> And please mirror friction to Codeberg for more wide audience.

No.

**Reply by ghost** [reply]
Thank a lot,
Please tell why "no"

**Reply by rodlie** [reply]
Short answer didn't work :)

Why would I mirror the project? That just creates fragmentation and adds more maintenance work.

Now I need to create a new user (on a new service that may or may not exist in one year), create a new project, mirror that, and interact with users on that platform.

Now we have two places where Friction is maintained, for what? So you (or someone else) can git clone from another url than github.com? or another place where users submit bugs?

At best I could agree to maybe do a pure git mirror, but again why? Source code (tarballs) and binaries are mirrored on SourceForge already.

In the end I don't want to add more work for no perceived gain (IMHO).

**Reply by ghost** [reply]
Some people do not like to use GitHub.  When more easy to connect developers, people support more. I understand it's hard to maintain. But some advantages there for users and developers. Thanks for answer. Good days...
